source/iostreams.tex

```diff
@@ -2633,7 +2633,7 @@
 the function signature
 \tcode{basic_ostream\& stream::operator\shl(ios_base\& (*)(ios_base\&))}
 to permit expressions of the form
-\tcode{cout \shl dec}
+\tcode{cout \shl{} dec}
```

Comment:
I did an "appearance" check with diffpdf, and no guillemets showed up. Maybe because the macros are only ever used inside \tcode and similar.